.github/workflows/release.yaml

```
@@ -9,7 +9,7 @@ on:
       bundle_channels:
         description: "Bundle channels"
         required: true
-        default: stable-1.0
+        default: "stable,stable-3.0"
```

OSSM 3.1 operator should include stable 3.1 channel, right?
```suggestion
        default: "stable,stable-3.0,stable-3.1"
```

---

Done